src/test/java/arcadia/constants/EndPoint.java

```diff
@@ -1,12 +1,20 @@
 package arcadia.constants;
 
 public enum EndPoint {
-    TRAINING("mercury1_21/"),
-    PROJECT("/index.lp?app=projects2&appoption=10&project=2&projName=projectName"),
+    TRAINING("mercury1_21_1/"),
+    PROJECT("/index.lp?app=projects2&appoption=10&project=2&projName=quickstart"),
```

Comment:
This change is not needed. The project name would be picked from environment variables.